packages/react-popover/src/PopoverTrappingFocus.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Popover, PopoverTrigger, PopoverSurface } from './index';
+import { Button } from './utils.stories';
+import { ExampleContent } from './utils.stories';
+
+export const TrappingFocus = () => (
```

Comment:
This is hard to play with due to an iframe design - if you try to navigate using the keyboard, the focus goes out of the iframe. This is the correct behaviour but hard to understand in the docsite. Unfortunately I do not have any idea how to make it better :/

Reply:
I think it gets back to the idea that we need our own docsite, but let's stay away from that until we have a more full set of components at least ;)